.github/workflows/ci.yaml

```diff
@@ -0,0 +1,94 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  cargo-test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Generate Cargo.lock so we can cache from it
+        run: cargo generate-lockfile
+      - uses: actions/cache@v2
```

Comment:
Do we need the workaround from https://github.com/IronCoreLabs/recrypt-rs/blob/master/.github/workflows/ci.yaml#L17? I believe it was failing to actually restore the cache without that workaround, in certain environments. The cache action would print a zillion warning messages in the Actions log, but it would still succeed.
If we don't need that here, then maybe we can remove it from the other repos where we use it.

Reply:
It should no longer be needed. They released a fix for the issue a few weeks ago.
We should be able to remove it from our other repos (and switch them all to cache@v2), but I assume it's not worth making all the PRs without other changes.

Reply:
Very cool.